src/scss/_mixins.scss

```diff
@@ -7,7 +7,7 @@
 
 @mixin button ($width: 110px, $height: 46px, $border-radius: 3px,) {
   @include fonts-style(16px, 700, 24px, $basic-state-text);
-  font-family: 'Poppins', sans-serif;
+  font-family: $font-family;
```

Comment:
Do you need to dublicate this property here? It's already provided in main styles.